include/openssl/opensslconf.h.in

```diff
@@ -9,11 +9,22 @@
  * https://www.openssl.org/source/license.html
  */
 
-#include <openssl/opensslv.h>
-
 #ifndef HEADER_OPENSSLCONF_H
 # define HEADER_OPENSSLCONF_H
 
+/*
+ * Important Note: Since the change of the versioning scheme in commit
+ * 3a63dbef15b6, the two header files
```

Comment:
I don't think it's a good idea to have commit ID's in the source; maybe a link to the PR?  Even that is not great, IMO.

Reply:
IMO, git is a distributed vcs, so every information should be available in the local repository, without needing an internet connection. For that reason I think good commit messages are more important than good pull request writeups, and that references should be commit ids, not GitHub links.

Reply:
I could add the pull request number in the comment, but not the link.

Reply:
Are most people who are looking at the header files just folks looking at a bundled distro release, or who are cloning the repo and building from source?

Reply:
> Are most people who are looking at the header files just folks looking at a bundled distro release, or who are cloning the repo and building from source?

The former type of people won't read the comment anyway.

But honestly, I always pay attention that the comment contains all essential information, nothing important was omitted. So there is no need to look up the commit message in order to understand the comment. But the reference to that commit is useful information nevertheless.  Would it look better for you, if the commit id was in parentheses, to make it look less important?

```
 * Important Note: Since the change of the versioning scheme (commit
 * 3a63dbef15b6), the two header files
```

Reply:
You don't need to please me. :)  I made a point, you disagree, that's okay.